src/group_theory/group_action/group.lean

```diff
@@ -155,6 +160,15 @@ commute.smul_right_iff (units.mk0 c hc)
   commute (c • a) b ↔ commute a b :=
 commute.smul_left_iff (units.mk0 c hc)
 
+protected lemma mul_action.bijective₀ (ha : a ≠ 0) : bijective (λ b : β, a • b) :=
+mul_action.bijective $ units.mk0 a ha
```

Comment:
```suggestion
protected lemma mul_action.bijective₀ (ha : a ≠ 0) : bijective ((•) a : β -> β) :=
mul_action.bijective $ units.mk0 a ha
```
etc is better when used in the context of `apply f_injective` as you don't need to follow up with a `dsimp`.